docker-compose.yml

```
@@ -12,7 +12,7 @@ services:
   db:
     image: postgres:14.4
     ports:
-      - "5432"
+      - "5432:5432"
```

```suggestion
      - "5432"
```
Do not expose this port in prod